config/deploy_development.env

```diff
@@ -1,5 +1,5 @@
-NYPL_API_SCHEMA_URL=https://platform.nypl.org/api/v0.1/current-schemas/Bib
-MLN_API_URL=https://dev-platform.nypl.org/api/v0.1/mylibrarynyc/teacher-sets
+NYPL_API_SCHEMA_URL=https://dev-platform.nypl.org/api/v0.1/current-schemas/Item
+MLN_API_URL=http://my-library-nyc-app-development.us-east-1.elasticbeanstalk.com/api/v0.1/mylibrarynyc/item-availability
```

Comment:
**are we going add this api in dev-platform docs?

https://dev-platform.nypl.org/api/v0.1/mylibrarynyc/item-availability need to access like this right?**